app/models/contact_form.rb

```diff
@@ -0,0 +1,16 @@
+class ContactForm < MailForm::Base
+  attribute :name,     validates: true
+  attribute :email,    validates: /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/i
+  attribute :message
+
+  def headers
+    {
+      to: 'homesshelter@gmail.com',
+      from: %("#{name}" <#{email}>)
+    }
+  end
+
+  def self.send_message(content)
+    new(name: content[:name], email: content[:email], message: content[:message]).deliver
```

Comment:
@jtruong2 Very cool use of this `mailer` gem, nicely done ensuring that only `:name`, `:email`, `:message` are passed through the params in the controller